flink-java/src/main/java/org/apache/flink/api/java/utils/ParameterTool.java

```diff
@@ -124,6 +138,13 @@ else if(arg.startsWith("-")) {
 		return fromMap(map);
 	}
 
+    /**
+     * Returns {@link ParameterTool} for the given properties file
```

Comment:
Maybe it would make sense to link to the documentation of Java Properties (so that people know how the format looks like)